get_only_massspecgym.py

```
@@ -78,52 +79,59 @@ def remove_duplicates_array(array):
+        FilteringMassSpecGym.filter_massspecgym(
+            molecule_pairs_train,
+            path_pairs=config.PREPROCESSING_DIR_TRAIN,
+            config=config,
+            prefix="ed_mces_indexes_tani_incremental_train",
+        )
+    )
 
 else:
-    molecule_pairs_train_filtered=molecule_pairs_train
+    molecule_pairs_train_filtered = molecule_pairs_train
     indexes_tani_multitasking_train_filtered = indexes_tani_multitasking_train
```

Ruff (0.14.5)

Undefined name `indexes_tani_multitasking_train`

(F821)